modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java

```diff
@@ -250,16 +252,14 @@ public void testApplicationCreationWithCustomAttributes(String applicationName,
     @AfterClass(alwaysRun = true)
     public void destroy() throws Exception {
         removeAllApps();
-        super.cleanUp();
     }
 
     public void removeAllApps() throws Exception {
         //delete created applications
-        HttpResponse getAllAppResponse = restAPIStore.getAllApp();
-        String[] array = getAllAppResponse.getData().replace("[", "")
-                .replace("]", "").split(",");
-        for (int app = 0; (array.length - 1) > app; app++) {
-            restAPIStore.deleteApplication(array[app]);
+        ApplicationListDTO getAllAppResponse = restAPIStore.getAllApps();
+        List<ApplicationInfoDTO> array = getAllAppResponse.getList();
```

Comment:
Shall we rename this to applicationDTOList or something similar?

Reply:
Fixed